cli/src/main/kotlin/uk/gov/justice/digital/cli/command/PreviewDomain.kt

```diff
@@ -53,11 +53,19 @@ class PreviewDomain(private val service: DomainService) : Runnable {
     override fun run() =
         runAndHandleExceptions(parent) {
             val result = service.previewDomain(domainName(), domainStatus, displayHeight())
-            val output = listOf(
-                "\n@|bold,green Previewing domain ${domainName()} with status ${domainStatus}|@\n",
-                generateOutput(result),
-                ""
-            ).joinToString("\n")
+            val output =
+                if (result.size > 1)
```

Comment:
What would we want the behaviour to be if result is 1?

Reply:
It would be treated as an empty result set which is what we want. The response format is essentially ```
listOf(
    listOf("heading1", "heading2"......"headingn"),
    listOf( ... data row 1 ),
    listOf( ... data row n),
)```